src/shipit_pulse_listener/shipit_pulse_listener/listener.py

```diff
@@ -34,11 +34,14 @@ def __init__(self, configuration):
         # Start on first page
         self.before = None
 
+        # Limit to diffs on last 12 hours
+        self.date_limit = datetime.now() - timedelta(hours=12)
```

Comment:
Is there no limit on the diffs returned by the phabricator API?

Reply:
I get results by batches of 10 diffs, but also use the pagination to get the whole list.